### PR TITLE
Update fix-issues-with-printers-scanners-and-lob-applications-that-se…

### PR DESCRIPTION
**Pull request by arindamthokder**
…nd-email-using-off.md

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 1]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
 
 3. Check that your device or application supports TLS version 1.2 or above. The best way to check is by upgrading the firmware on the device or updating the application you're sending email from to the latest version. Contact your device manufacturer to confirm that it supports TLS version 1.2 or above.
 
+### Error: Authentication unsuccessful
+
+If you are getting one of the following erros-
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:119]:
```suggestion
If you are getting one of the following errors: 
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 2]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
 
 3. Check that your device or application supports TLS version 1.2 or above. The best way to check is by upgrading the firmware on the device or updating the application you're sending email from to the latest version. Contact your device manufacturer to confirm that it supports TLS version 1.2 or above.
 
+### Error: Authentication unsuccessful
+
+If you are getting one of the following erros- 
+
+535 5.7.3 Authentication unsuccessful
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:121]:
```suggestion
- 535 5.7.3 Authentication unsuccessful
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 3]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+5.7.57 Client not authenticated to send mail
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:122]:
```suggestion
- 5.7.57 Client not authenticated to send mail
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 4]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+
+There are a few things you should check:
+
+1. Enable Client SMTP submission on the licensed mailbox being used:
+   From Microsoft 365 Admin Center, go to Active Users and select the user
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:127]:
```suggestion
   From Microsoft 365 Admin Center, go to Active Users and select the user.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 5]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   Go to Mail tab
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:128]:
```suggestion
   Go to the Mail tab.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 6]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   In the “Email apps” section, click on “Manage email apps”
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:129]:
```suggestion
   In the “Email apps” section, select **Manage email apps**.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 7]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   Verify the “Authenticated SMTP” setting is checked (enabled)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:130]:
```suggestion
   Verify that the “Authenticated SMTP” setting is checked (enabled).
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 8]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   Click Save changes.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:131]:
```suggestion
   Select **Save changes**.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 9]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   
+2. Disable Multi Factor Authentication (MFA) on the licensed mailbox being used:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:133]:
```suggestion
2. Disable Multi-Factor Authentication (MFA) on the licensed mailbox being used:
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 10]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   In the Microsoft 365 Admin Center, in the left navigation menu choose Users > Active users.
+   On the Active users page, choose Multi-factor authentication.
+   On the multi-factor authentication page, select the user and disable the Multi-Factor auth status.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:136]:
```suggestion
   - In the Microsoft 365 admin center, in the left navigation menu, choose **Users** > **Active users**.
   - On the Active users page, choose **Multi-Factor Authentication**.
   - On the Multi-Factor Authentication page, select the user and disable the Multi-Factor Authentication status.
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 11]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   
+3. Disable the [Azure Security Defaults](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/concept-fundamentals-security-defaults) by toggling the “Enable Security Defaults” to “No”:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:138]:
```suggestion
3. Disable the [Azure Security Defaults](/azure/active-directory/fundamentals/concept-fundamentals-security-defaults) by toggling the **Enable Security Defaults** to **No**:
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 12]
```diff
@@ -114,6 +114,34 @@ This can be caused by a number of issues:
...
+   Sign in to the Azure portal as a Security administrator, Conditional Access administrator, or Global administrator.
+   Browse to Azure Active Directory > Properties.
+   Select Manage security defaults.
+   Set the Enable security defaults toggle to No.
+   Select Save.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:143]:
```suggestion
   - Sign in to the Azure portal as a Security administrator, Conditional Access administrator, or Global administrator.
   - Browse to Azure Active Directory > Properties.
   - Select **Manage security defaults**.
   - Set the **Enable security defaults** toggle to **No**.
   - Select **Save**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arindamthokder, kindly commit my suggested changes. Many thanks.

**Comment by arindamthokder** [comment]
> Copy review - @arindamthokder, kindly commit my suggested changes. Many thanks.

Suggested changes committed. Thanks a lot.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**